plugins/fastify-as-http/package.json

```diff
@@ -0,0 +1,34 @@
+{
+        "name": "@godspeedsystems/plugins-fastify-as-eventsource",
+        "version": "1.0.0",
+        "description": "fastify as eventsource plugin for Godspeed Framework",
+        "publishConfig": {
+                "access": "public"
+        },
+        "main": "dist/index.js",
+        "types": "dist/index.d.js",
+        "scripts": {
+                "dev": "tsc --watch",
+                "build": "tsc",
+                "prepublishOnly": "npm run build"
+        },
+        "license": "ISC",
+        "devDependencies": {
+                "@types/lodash": "^4.14.202",
+                "@types/qs": "^6.9.10",
+		"@types/fastify-jwt": "^0.8.1",
+                "lodash": "^4.17.21",
```

Comment:
i did not find use of lodash anywhere in your changes @pavan-godspeed

Reply:
removed unwanted (lodash) package from package.json.